packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1913,7 +1913,7 @@ export const COMPONENT_LOCALE_ZH_CN: ComponentLocale;
 export const COMPONENT_LOCALE_ZH_TW: ComponentLocale;
 
 // @public
-export type ComponentIcons = Record<keyof typeof DEFAULT_COMPONENT_ICONS, JSX.Element>;
+export type ComponentIcons = Record<keyof typeof DEFAULT_COMPONENT_ICONS, React_2.JSX.Element>;
```

Comment:
I'm wondering if `React_2.` can be deleted here too

Reply:
This is a generated file.

Reply:
Yep,  just noticed that React_2 was deleted in most of the places and wanted to check if we can do anything with this  one too